packages/backend/src/open_payments/payment/incoming/routes.test.ts

```diff
@@ -421,4 +424,171 @@ describe('Incoming Payment Routes', (): void => {
+          accountId: account.id,
+          description: '2nd incoming payment',
+          expiresAt,
+          incomingAmount: {
+            value: BigInt(321),
+            assetCode: asset.code,
+            assetScale: asset.scale
+          },
+          externalRef: '#321'
+        })) as IncomingPayment
+        incomingPayment3 = (await incomingPaymentService.create({
+          accountId: account.id,
+          description: '3rd incoming payment',
+          expiresAt,
+          incomingAmount: {
+            value: BigInt(213),
+            assetCode: asset.code,
+            assetScale: asset.scale
+          },
+          externalRef: '#213'
+        })) as IncomingPayment
+        paymentIds = [
+          incomingPayment.id,
+          incomingPayment2.id,
+          incomingPayment3.id
+        ]
+        result = [
+          {
+            id: `https://wallet.example/${account.id}/incoming-payments/${incomingPayment.id}`,
+            accountId: `https://wallet.example/${account.id}`,
+            incomingAmount: {
+              value: '123',
+              assetCode: asset.code,
+              assetScale: asset.scale
+            },
+            description: incomingPayment.description,
+            expiresAt: expiresAt.toISOString(),
+            receivedAmount: {
+              value: '0',
+              assetCode: asset.code,
+              assetScale: asset.scale
+            },
+            externalRef: '#123',
+            state: IncomingPaymentState.Pending.toLowerCase()
+          },
+          {
+            id: `https://wallet.example/${account.id}/incoming-payments/${incomingPayment2.id}`,
+            accountId: `https://wallet.example/${account.id}`,
+            incomingAmount: {
+              value: '321',
+              assetCode: asset.code,
+              assetScale: asset.scale
+            },
+            description: incomingPayment2.description,
+            expiresAt: expiresAt.toISOString(),
+            receivedAmount: {
+              value: '0',
+              assetCode: asset.code,
+              assetScale: asset.scale
+            },
+            externalRef: '#321',
+            state: IncomingPaymentState.Pending.toLowerCase()
+          },
+          {
+            id: `https://wallet.example/${account.id}/incoming-payments/${incomingPayment3.id}`,
+            accountId: `https://wallet.example/${account.id}`,
+            incomingAmount: {
+              value: '213',
+              assetCode: asset.code,
+              assetScale: asset.scale
+            },
+            description: incomingPayment3.description,
+            expiresAt: expiresAt.toISOString(),
+            receivedAmount: {
+              value: '0',
+              assetCode: asset.code,
+              assetScale: asset.scale
+            },
+            externalRef: '#213',
+            state: IncomingPaymentState.Pending.toLowerCase()
+          }
+        ]
+      }
+    )
+    test.each`
```

Comment:
Can things be rearranged so the `beforeEach` creating the incoming payments isn't run before all of these expected failure tests?